### DISCUSSION TITLE
Don't know how to access Management API from Postman

### DISCUSSION THREAD

**Discussion by matthewtusker** [zitadel/zitadel#5704] [General]
Hi, I'm trying to test the management API in a locally running instance of Zitadel but I can't figure out how to authorise Postman to use the API.

The [Management API Docs](https://zitadel.com/docs/apis/mgmt/management-api) says I can use OAuth 2.0, and I've configured Postman with these settings, but I can't get it to work with PKCE, and the instance contains no details about the secret (which it shouldn't) or how to reset it so that I can get a new one. I've tried Basic auth with the default User (`zitadel-admin@zitadel.localhost`), but get an invalid auth header error (`invalid auth header (AUTH-7fs1e)`).

I'm not sure where to go from here. I need to be able to test importing users from an existing system, and so far I haven't been able to get access to the API to prove that it will work.

**Comment by hifabienne** [comment]
Hei @matthewtusker 
I don't use Postman, I use insomnia but I guess it should work similar.
What I have configured:
Grant Type: Auhtorization Code
Authorization URL: $YOUR_DOMAIN/oauth/v2/authorize
Token URL: $YOUR_DOMAIN/oauth/v2/token
Client ID: I use the ZITADEL Console applications client id, but you can also take another app. (User Agent) 
Use PKCE: enabled
Redirect URL: $YOUR_DOMAIN/ui/console/auth/callback
scopes: openid profile (make sure to also include zitadel project scope, if you don't use the zitadel console app: urn:zitadel:iam:org:project:id:zitadel:aud)

Let me know if that already helps

**Reply by matthewtusker** [reply]
Ah ha! It's working now. Thanks for the help. I don't understand what the docs were trying to tell me, is that page still valid? It doesn't mention using the Console client to access the API, unless I've missed a page somewhere.

**Reply by hifabienne** [reply]
You can also take another application. In that case you just have to include the scope urn:zitadel:iam:org:project🆔zitadel:aud

**Comment by matthewtusker** [comment]
Thanks again for the help, I've managed to confirm the functionality we need for our app so should be happy to go all in on a self-hosted Zitadel instance! 🎉
